src/components/organisms/EntityDetails/EntityDetailsLayer.tsx

```diff
@@ -206,8 +212,6 @@ const EntityDetailsLayer: FC<PropsWithChildren<EntityDetailsLayerProps>> = ({
     safeRefetch(refetch);
   }, [entity, daysFilterValue]);
 
-  useInterval(() => safeRefetch(refetchMetrics), 2000);
```

Comment:
This should rather stay. As long as `skip` condition is truthy, it will rather not make a request. If it's not truthy, we want it to be updated.